application/laravel/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM composer:latest
```

```suggestion
FROM composer:1.10.10
```

Let's pin the base image version.